web/src/components/pages/contribution/report/report.css

```diff
@@ -61,6 +61,8 @@
         border-radius: 2px;
         padding: 10px;
         width: 100%;
+        margin-top: 8;
+        max-width: 115%;
```

Comment:
Can you please explain where `115%` came from?

Margin change seems fine to me.

Reply:
Since the width of TextArea is set to 100% and there should be the maximum width of textArea to be set to 115%.
We can also use `resize: vertical` to solve #2587 which will fix the width length and we can expand vertically only.
@rileyjshaw Your views?

Reply:
Recommendations:

- [ ] Delete lines 46-48 above. The `width` constraint on `.detail` is why you needed to overflow the container boundaries with `width: 115%`.
- [ ] Delete `margin-top: 8`. It has no units, so it's not doing anything. And the `textarea` already has an inline style of `margin-top: 8px` :)
- [ ] Change the `textarea`'s `width` to `calc(100% - 60px)`, and `max-width` to `100%`. This preserves the initial look, and constrains the element within our layout.

If you update this PR, I'll re-review and land it. Thanks for your attention to detail in noticing this bug.

---

Also, a note on communication. On this issue and several others, you have pinged either myself or other maintainers directly. Several times. Please do not do that. Please also refrain from sending us personal emails, direct messages, etc.

We love our community, and we do our best to be helpful and responsive. We get a notification for every pull-request and comment. We have noticed your PRs, and have scheduled time to review them.

Good feedback takes time and attention, especially since you've submitted features that weren't previously in our issue queue or roadmap. @-mentioning us this often distracts us from other areas of the project / community that also require attention.

Thanks for your understanding, and I hope the code feedback helped. Comment in-thread if you have further questions.